activestorage/app/models/active_storage/preview.rb

```diff
@@ -75,6 +75,10 @@ def key
     end
   end
 
+  # Downloads the file associated with this preview's variant. Raises ActiveStorage::Preview::UnprocessedError if the
```

Comment:
Could you wrap the line at 80 characters? Or maybe add a linebreak after the first sentence?
```suggestion
  # Downloads the file associated with this preview's variant.
  # Raises ActiveStorage::Preview::UnprocessedError if the
```

Reply:
I added a line break after the first sentence.